### PR TITLE
Adds revision to project list row items

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
link #5054 

Screen Shot 2019-10-23 at 5 44 17 PM

In action:
copy_rev

This PR introduces a new component ClipboardCopyButton which is a stripped down version of https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopyButton.tsx.  Patternfly has a ClipboardCopy component but it's tethered to an input which doesn't really make sense in this case.  We can always replace our implementation with something from PF in the future if that becomes an option.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

File: awx/ui_next/src/components/ClipboardCopyButton/ClipboardCopyButton.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    background-color: #0066cc;
+    color: white;
+  }
+`;
+
+export const clipboardCopyFunc = (event, text) => {
+  const clipboard = event.currentTarget.parentElement;
+  const el = document.createElement('input');
+  el.value = text;
+  clipboard.appendChild(el);
```
Comment (keithjgrant) [inline awx/ui_next/src/components/ClipboardCopyButton/ClipboardCopyButton.jsx:22]: I think you could theoretically just append this to `document.body` and not need the `event` parameter passed in to the function.

**Review by keithjgrant — APPROVED** [review]
Looks good to me!

**Comment by jakemcdermott** [comment]
recheck

**Comment by jakemcdermott** [comment]
regate

**Pull request merged**